_posts/2020-11-30-dataops-dynamodb-to-snowflake.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: post
+title: Batch ELT from AWS DynamoDB to Snowflake
+date: 2020-11-30
+tags: snowflake dynamodb aws data elt s3
+author: Paul Symons
+image: img/blog/dynamodb-to-snowflake/logo.jpg
+---
+<style type="text/css">
+  @media only screen and (min-width: 1000px) {
+    h2, h3 {
+      margin-top:1em;
+    }
+    div.wrapper { 
+      max-width:900px;
+    }
+  }
+</style>
+
+In this article we are going to discuss a new way to batch load your DynamoDB tables directly into Snowflake without proprietary tools. But first...
```

Comment:
Suggestion; do we need more of a context/hook as to "why" someone would want to do that?

Reply:
i.e. what business/real-world problem does this technical solution solve?

Reply:
Maybe. How would you describe what real world problem this might solve?

Reply:
I might describe it as a fictional example / scenario. For example, a weather product where the team also wanted to run some ad-hoc analysis on the live data, etc (which DDB is not suitable for).